redux.d.ts

```diff
@@ -46,7 +46,7 @@ declare module redux {
 		(createStore: ICreateStoreGeneric<TState>): ICreateStoreGeneric<TState>;
 	}
 
-	export function createStore<TState>(reducer: IReducer<TState>, initialState?: TState): IStore<TState>;
+	export function createStore<TState>(reducer: IReducer<TState>, initialState?: TState, enhancer?: Function): IStore<TState>;
```

Comment:
IMO `Function` should be enough here.

BTW: there should be second definition for this method:

```
export function createStore<TState>(reducer: IReducer<TState>, enhancer?: Function): IStore<TState>;
```